### PR TITLE
feat: add components & pages

### PR DESCRIPTION
**Pull request by anik31**
## Pages added - 
- Home/Feed
- Explore
- Bookmarks
- Profile
- Auth - Login, signup
- Modals - create post, edit profile

## Files to check
- src/App.js
- src/components/card/DisplayPost.jsx
- src/components/card/DisplayProfile.jsx
- src/components/follow/Follow.jsx
- src/components/modal/CreatePost.jsx
- src/components/modal/EditProfile.jsx
- src/components/navbar/Navbar.jsx
- src/components/notFound/NotFound.jsx
- src/components/routes/WithSidebar.jsx
- src/components/sidenav/Sidenav.jsx
- src/pages/auth/Login.jsx
- src/pages/auth/Signup.jsx

 _Note - Ignore other files._

### CODE REVIEW

File: src/components/follow/Follow.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Box, Text, Button, Stack } from "@chakra-ui/react";
+import { Divider } from '@chakra-ui/react';
+import { Avatar } from '@chakra-ui/react';
```
Comment (Maianki) [inline src/components/follow/Follow.jsx]: You can import all of these in single line.

**Review by Maianki — APPROVED** [review]
Great job. The code looks clean. Good to see you are using chakra UI. Has added one suggestion. Everything else looks good.

**Review by abhisheksahani-nova — APPROVED** [review]
Looks clean and good to merge, good UI

**Review by sainath7878 — APPROVED** [review]
UI is awesome! Code looks very neat and understandable. Great job!!

File: src/components/follow/Follow.jsx (hunk 1 above)
Comment (anik31) [reply src/components/follow/Follow.jsx]: done

**Pull request merged**